developer/testing/creating-test-data.md

```diff
@@ -2,14 +2,134 @@
 
 For Meteor integration tests, see [Creating Test Data for Meteor Tests](/developer/testing/creating-test-data-meteor.md).
 
-For non-Meteor Jest tests, we do not yet have a factory solution that works outside of Meteor, but you may import and use `faker` as necessary. Coming soon!
+For non-Meteor Jest tests, Reaction Commerce provides a simple data `Factory` test utility that can create mock data for unit and integration test. The test util uses [@reactioncommerce/data-factory](https://github.com/reactioncommerce/data-factory) to attach all core schemas to the `Factory` object.
 
-## Faker
+## Creating mock data
+Creating mock data structures.
+``` js
+import { Factory } from "/imports/test-utils/helpers/factory";
+const mockTag = Factory.Tags.makeOne();
```

Comment:
Instead of commenting the mockTag output out, put them in a separate code block:

```js
import ..
const mockTag = Factory.Tags.makeOne();
```
The `mockTag` output returns an object with fake data like this:
```js
{
   _id: "..."
}
```